packs/ibmmq/dashboards/ibmmq.json

```diff
@@ -0,0 +1,224 @@
+{
+  "name": "IBM MQ PROD",
+  "description": null,
```

Comment:
Can you please update all description fields to empty strings `""`, so the schema checks don't fail.
We are working to improve this but we unfortunately have to do this manually now

Reply:
Fixed! @polfliet